packages/utilities/psammead-styles/src/font-faces.js

```diff
@@ -0,0 +1,94 @@
+const baseFontUrl = 'https://ws-downloads.files.bbci.co.uk/fonts/';
+
+export const persian = {
+  sansBold: `@font-face {
+    font-family: NassimPersian, Arial, Verdana,	Geneva,	Helvetica, sans serif;
+    font-weight: 700;
+    font-style: normal;
+    src: url('${baseFontUrl}NassimPersian/latest/bold.woff') format('woff'), url('${baseFontUrl}NassimPersian/latest/bold.eot') format('eot'), url('${baseFontUrl}NassimPersian/latest/bold.ttf') format('ttf');
+    font-display: optional;
+  }`,
+  sansRegular: `@font-face {
+    font-family: NassimPersian, Arial, Verdana,	Geneva,	Helvetica, sans serif;
+    font-weight: 400;
+    font-style: normal;
+    src: url('${baseFontUrl}NassimPersian/latest/normal.woff') format('woff'), url('${baseFontUrl}NassimPersian/latest/normal.eot') format('eot'), url('${baseFontUrl}NassimPersian/latest/normal.ttf') format('ttf');
+    font-display: optional;
+  }`,
+};
+
+export const arabic = {
+  sansBold: `@font-face {
+    font-family: NassimArabic, Arial, Verdana,	Geneva,	Helvetica, sans serif;
+    font-weight: 700;
+    font-style: normal;
+    src: url('${baseFontUrl}NassimArabic/latest/bold.woff') format('woff'), url('${baseFontUrl}NassimArabic/latest/bold.eot') format('eot'), url('${baseFontUrl}NassimArabic/latest/bold.ttf') format('ttf');
+    font-display: optional;
+  }`,
+  sansRegular: `@font-face {
+    font-family: NassimArabic, Arial, Verdana,	Geneva,	Helvetica, sans serif;
+    font-weight: 400;
+    font-style: normal;
+    src: url('${baseFontUrl}NassimArabic/latest/normal.woff') format('woff'), url('${baseFontUrl}NassimArabic/latest/normal.eot') format('eot'), url('${baseFontUrl}NassimArabic/latest/normal.ttf') format('ttf');
+    font-display: optional;
+  }`,
+};
+
+export const news = {
+  sansBold: `@font-face {
+    font-family: "ReithSans";
+    font-weight: 700;
+    font-style: normal;
+    src: url('${baseFontUrl}ReithSans/latest/bold.woff2') format('woff2'), url('${baseFontUrl}ReithSans/latest/bold.woff') format('woff');
```

Comment:
We should continue to use the gel base url for news, it has better cross over with other UK sites like sport